pkg/lib/angular-dialog.js

```diff
@@ -70,6 +70,29 @@
             return {
                 restrict: 'E',
                 transclude: true,
+                template: '<modal-group><ng-transclude></ng-transclude></modal-group>',
+                link: function(scope, element, attrs) {
+
+                    scope.cancel = function() {
+                        scope.$dismiss();
+                    };
+
+                    scope.result = function(success, result) {
+                        /* Close dialog on success */
+                        if (success)
+                            scope.$close(result);
+                    };
+                },
+            };
+        }
+    ])
+
+    .directive('modalGroup', [
```

Comment:
What does modalGroup do. Perhaps a multiline comment here would be helpful describing this directive.